libcontainer/cgroups/systemd/user.go

```diff
@@ -91,16 +90,5 @@ func DetectUserDbusSessionBusAddress() (string, error) {
 			return busAddress, nil
 		}
```

Comment:
When `$DBUS_SESSION_BUS_ADDRESS` is unset but `$XDG_RUNTIME_DIR` is set, probably we can try `systemctl --user --no-pager show-environment`